### ISSUE TITLE
SElinux prevents bootstrap VM to start

### ISSUE THREAD

**Issue by tahonen** [RedHat-EMEA-SSA-Team/hetzner-ocp4#50]
When running on RHEL 7.7 and SELinux is in Enforcing mode bootstrap VM does not start.
```
TASK [openshift-4-cluster : Start VirtualMachine iot-bootstrap] ***************************************************************************************************************************************************
Tuesday 01 October 2019  14:34:13 +0300 (0:00:00.404)       0:17:06.550 ******* 
An exception occurred during task execution. To see the full traceback, use -vvv. The error was: libvirtError: internal error: qemu unexpectedly closed the monitor: 2019-10-01T11:34:15.354797Z qemu-kvm: -fw_cfg name=opt/com.coreos/config,file=/var/lib/libvirt/images/iot-bootstrap.ign: can't load /var/lib/libvirt/images/iot-bootstrap.ign
fatal: [localhost]: FAILED! => {"changed": false, "msg": "internal error: qemu unexpectedly closed the monitor: 2019-10-01T11:34:15.354797Z qemu-kvm: -fw_cfg name=opt/com.coreos/config,file=/var/lib/libvirt/images/iot-bootstrap.ign: can't load /var/lib/libvirt/images/iot-bootstrap.ign"}
```

**Comment by flyemsafe** [comment]
Does your label looks like this?

```
admin@lunchbox01:~ [s1] $ sudo ls -lsaFZ /var/lib/libvirt/ | grep image
drwx--x--x. root root system_u:object_r:virt_image_t:s0 images/
```

**Comment by tahonen** [comment]
[root@srv-31 hetzner-ocp4]# ls -lsaFZ /var/lib/libvirt/ | grep image
drwx--x--x. root root system_u:object_r:virt_image_t:s0 images/

**Comment by tahonen** [comment]
Added svirt_image_to too bootstrap.ign and that worked

[root@srv-31 hetzner-ocp4]# ls -lsaFZ /var/lib/libvirt/images/ | grep image
drwx--x--x. root root system_u:object_r:virt_image_t:s0 ./
-rw-r--r--. root root system_u:object_r:svirt_image_t:s0 iot-bootstrap.ign
-rw-r--r--. qemu qemu system_u:object_r:svirt_image_t:s0:c72,c969 iot-bootstrap.qcow2
-rw-r--r--. root root system_u:object_r:virt_image_t:s0 iot-compute-0.ign
-rw-r--r--. qemu qemu system_u:object_r:svirt_image_t:s0:c156,c171 iot-compute-0.qcow2
-rw-r--r--. root root system_u:object_r:virt_image_t:s0 iot-compute-1.ign
-rw-r--r--. qemu qemu system_u:object_r:svirt_image_t:s0:c375,c648 iot-compute-1.qcow2
-rw-r--r--. root root system_u:object_r:virt_image_t:s0 iot-compute-2.ign
-rw-r--r--. qemu qemu system_u:object_r:svirt_image_t:s0:c200,c316 iot-compute-2.qcow2
-rw-r--r--. root root system_u:object_r:virt_image_t:s0 iot-master-0.ign
-rw-r--r--. qemu qemu system_u:object_r:svirt_image_t:s0:c243,c362 iot-master-0.qcow2
-rw-r--r--. root root system_u:object_r:virt_image_t:s0 iot-master-1.ign
-rw-r--r--. qemu qemu system_u:object_r:svirt_image_t:s0:c360,c669 iot-master-1.qcow2
-rw-r--r--. root root system_u:object_r:virt_image_t:s0 iot-master-2.ign
-rw-r--r--. qemu qemu system_u:object_r:svirt_image_t:s0:c491,c573 iot-master-2.qcow2

**Comment by tahonen** [comment]
Needed to add same label also to compute-*.ign and master-*.ign files.

**Comment by tahonen** [comment]
Same happens also with fresh CentOS 7.7

**Comment by rbo** [comment]
Tested with RHEL 8 (Dev branch: https://github.com/rbo/hetzner-ocp4/tree/dev ) works perfectly. 

Without any SELinux adjustments, feels strange. Works out of the box...

**Comment by rbo** [comment]
Fixed with commit: https://github.com/rbo/hetzner-ocp4/commit/9908ed086c2e1309223690a19adcf20e9e3fa746 

Will be merged with PR #75
